src/main/java/lebron/controller/MainWindow.java

```diff
@@ -0,0 +1,70 @@
+package lebron.controller;
+
+import javafx.scene.control.Label;
```

Comment:
Isn't this inconsistent ordering of import statements? In other files you have added spaces between different types of import statements, here you haven't. Again not a significant issue, but remember that a consistent ordering of import statements makes it easier to browse the list and determine the dependencies when there are many imports.